src/test/java/teammates/ui/webapi/SubmitFeedbackResponsesActionTest.java

```diff
@@ -156,6 +201,19 @@ public void testAccessControl_studentSubmissionPastEndTime_shouldAllowIfBeforeDe
         verifyCanAccess(submissionParams);
     }
 
+    @Test
+    public void testAccessControl_feedbackQuestionDoesNotExist_shouldThrowEntityNotFoundException() throws Exception {
```

Comment:
This test was accidentally removed in another merge of a PR. Re-added it after a discussion with @ElHachem02.